app/views/quota.html

```diff
@@ -84,11 +84,11 @@ <h3 class="text-center">Memory <small>Limit</small></h3>
                         <tr ng-repeat="(resourceType, specMax) in quota.spec.quota.hard"
                             ng-if="resourceType !== 'resourcequotas'"
                             ng-class="{
-                              warning: (quota.status.total.used[resourceType] | usageValue) >= (quota.status.total.hard[resourceType] | usageValue)
+                              warning: isAtLimit(quota, resourceType)
```

Comment:
Here's what it might look like. Opinion?

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/21941659/d47e24fc-d997-11e6-8e7c-3a3d293f375b.png)

We need to humanize some additional quota resources.